content/_data/changelogs/weekly.yml

```
@@ -25003,6 +25003,47 @@
   - version: '2.481'
     date: 2024-10-15
     changes:
+      - type: rfe
+        category: rfe
+        pull: 9787
+        issue: 73813
+        authors:
+          - mawinter69
+        pr_title: "[JENKINS-73813] Show a notification when scheduling a build fails"
+        message: |-
+          Show a notification when scheduling a build fails.
+      - type: rfe
+        category: rfe
+        pull: 9833
+        authors:
+          - janfaracik
+        pr_title: "Refine content and appearance of the 'Edit View' screen"
+        message: |-
+          Refine content and appearance of the <strong>Edit View</strong> screen.
```

I think that we usually use the word 'page' rather than 'screen'.
```suggestion
          Refine content and appearance of the <strong>Edit View</strong> page.
```